### ISSUE TITLE
Allow different extrusion settings for different tools/nozzles

### ISSUE THREAD

**Issue by oliof** [GridSpace/grid-apps#124]
I am trying my first multi extruder slice with kiri:moto, setting the second extruder to print supports.

I can't find a way to set different temperatures for the second extruder than for the first. My support material requires a different printing temperature than the first extruder. Even for simple multi color prints I might need a different set temperature for the same kind of filament with a different color.

I would also like to be able to set a different extrusion speed and part cooling fan power for extruders individually. They might require completely different speeds for good visual appearance or mechanical stability / inter layer adhesion than other extruders.

**Comment by stewartoallen** [comment]
Makes sense. I'll look into ways to do this that don't complicate the UI for simpler use cases.
